Web/Edubase.Web.UI/Models/EstablishmentDetailViewModel.cs

```diff
@@ -29,7 +29,7 @@ public EstablishmentDetailViewModel(IExternalLookupService extService = null)
         {
             [(int) eLookupGroupType.SingleacademyTrust] = "Single-academy trust",
             [(int) eLookupGroupType.MultiacademyTrust] = "Academy trust",
-            [(int) eLookupGroupType.SecureSingleAcademyTrust] = "Academy trust",
+            [(int) eLookupGroupType.SecureSingleAcademyTrust] = "Secure academy trust",
```

Comment:
This just looked like it had the wrong text in it.

Reply:
This controls the label text on the screen next to the academy name when viewing / editing the establishment

Reply:
that's one for Mark I would think - if it needs to change would it not be 'Secure single-academy trust'

Reply:
waiting for response from Mark

Reply:
Mark confirmed this is ok as is labelled "Academy trust"

Reply:
original value reverted